run-e2e.sh

```diff
@@ -29,7 +29,7 @@ if [[ "${PERF_TESTS_PRINT_COMMIT_HISTORY:-false}" == "true" ]]; then
   # The output file location assumes the script works inside a prow job.
   output_file="/workspace/_artifacts/perf-tests.gitlog"
   echo "k8s.io/perf-tests git log:" | tee $output_file || true
-  git log -n 10 --format="%H - %ad (%s)" --date=local | tee -a $output_file || true
+  git -C /go/src/k8s.io/perf-tests log -n 10 --format="%H - %ad (%s)" --date=local | tee -a $output_file || true
```

Comment:
Nice catch! Could you also remove lines 16-17? :)

Reply:
Done.